tgui-next/packages/tgui/styles/themes/syndicate.scss

```diff
@@ -45,6 +45,8 @@
   }
 }
 
+$tooltip-color-background: #4a0202;
+
```

Comment:
Theming has changed recently.

1. Import `Tooltip.scss` (this way you instruct Sass to compile tooltip with your custom styles).
2. Configure Tooltip using `$with` parameter.